hpat/tests/__init__.py

```diff
@@ -16,3 +16,6 @@
 from hpat.tests.test_io import *
 
 from hpat.tests.test_hpat_jit import *
+
+# performance tests
+import hpat.tests.tests_perf
```

Comment:
As far as I understand this line push performance testing during regular procedure.
We have to avoid this.
Need to find an other initialization method or exclude this module from unittest at initialization.

Reply:
Such import lets us run performance testing only via specific command:
`python -m hpat.runtests hpat.tests.tests_perf`

The following commands won't run performance testing:
```
python -m hpat.runtests
python -m hpat.runtests hpat.tests
```